frontends/web/src/routes/account/summary/balancerow.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, RenderableProps} from 'preact';
+import { Amount, FiatConversion } from '../../../components/rates/rates';
+import { translate, TranslateProps } from '../../../decorators/translate';
+
+interface ProvidedProps {
+    name: string;
+    balance: Amount;
+}
+
+type Props = ProvidedProps & TranslateProps;
+
+const BalanceRow = ({ name, balance }: RenderableProps<Props>) => {
```

Comment:
Is there a reason why you don't write `function BalanceRow({ name, balance }: RenderableProps<Props>) {`? I don't care too much about this but we seem to be somewhat inconsistent with this when compared to, for example, `update.tsx`.